test/passing/open-auto.ml.ref

```diff
@@ -31,10 +31,10 @@ let _ =
   y
 
 let () =
-  ( (let open Term in
-    term_result
-      ( const Phases.phase1 $ arch $ hub_id $ build_dir $ logs_dir
-      $ setup_logs ))
+  ( Term.(
+      term_result
+        ( const Phases.phase1 $ arch $ hub_id $ build_dir $ logs_dir
+        $ setup_logs ))
```

Comment:
This is correct, I think. I wonder which change caused this ?

Reply:
Before there were some cases where the syntax changed depended on whether it fitted on a single line or not (if it didn't fit we used the `let open` syntax in this case). Now it only depends on the shape of the expression so the `.` syntax is used for Pexp_apply.

Personally I found it weird to have the syntax change when the line became longer.

Reply:
I agree.